s4e-backend/src/main/java/pl/cyfronet/s4e/admin/product/AdminProductController.java

```diff
@@ -25,6 +28,70 @@
 public class AdminProductController {
     private final ProductService productService;
     private final AdminProductMapper adminProductMapper;
+    private final ProductCategoryRepository productCategoryRepository;
+
+    @Operation(summary = "Create Product Categories")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "OK"),
+            @ApiResponse(responseCode = "401", description = "Unauthenticated", content = @Content),
+            @ApiResponse(responseCode = "403", description = "Forbidden", content = @Content)
+    })
+    @PostMapping("/categories")
+    public void createCategories() {
+        productCategoryRepository.deleteAll();
```

Comment:
Aren't you deleting the default category by doing this?

Reply:
I was considering seeding the default category inside of REST request due to setting column default with hibernate decorator, but if it doesn't work I will skip default category deletion.